csfieldguide/templates/interactives/data-visualisation.html

```diff
@@ -0,0 +1,67 @@
+{% extends interactive_mode_template %}
+
+{% load i18n %}
+{% load static %}
+
+{% block html %}
+  <div class="container-fluid my-3 text-center">
+    <div id="data-vis-title" class="col-12">{% trans 'Data Visualisation' %}</div>
+    <div id="data-vis-instructions" class="col-12">
+      <div class="col-12">
+        {% blocktrans %}
+        Find the mode (most common number) of the data below.
+        You will have a limited amount of time to view the data.
+        {% endblocktrans %}
+      </div>
+      <button id="button-start" type="button" class="btn btn-success my-1">{% trans 'Start' %}</button>
+    </div>
+    <div id="data-vis-guesser" class="col-12 d-none">
+      <label for="data-vis-selector">{% trans 'What was the mode (most common number) of the data?' %}</label>
+      <form class="form-inline justify-content-center">
+        <select id="data-vis-selector" class="form-control text-center d-inline mx-2" size="1">
+          <option>0</option>
+          <option>1</option>
+          <option>2</option>
+          <option>3</option>
+          <option>4</option>
+          <option>5</option>
+          <option>6</option>
+          <option>7</option>
+          <option>8</option>
+          <option>9</option>
+        </select>
+        <button id="button-submit" type="button" class="btn btn-secondary d-inline mx-2">{% trans 'Submit' %}</button>
+      </form>
+    </div>
+    <div id="data-vis-next" class="col-12 my-1 d-none">
+      <button id="button-next" type="button" class="btn btn-secondary d-inline mx-2">{% trans 'Continue' %}</button>
+      <button id="button-quit" type="button" class="btn btn-danger d-inline mx-2">{% trans 'End Game' %}</button>
+    </div>
+    <div id="data-vis-restart" class="col-12 my-1 d-none">
+      <button id="button-restart" type="button" class="btn btn-primary d-inline mx-2">{% trans 'Restart Game' %}</button>
+    </div>
+    <div id="data-vis-game"class="col-12 d-none">
+      <table id="data-vis-grid" class="my-3"></table>
+      <canvas id="data-vis-barchart" height="400" width="800" role="img" aria-label="Bar chart showing how many times each value appears in the data."></canvas>
+      <canvas id="data-vis-piechart" height="400" width="600" role="img" aria-label="Pie chart showing the relative proportions of each value in the data."></canvas>
+    </div>
+    <div id="data-vis-results"class="col-12 d-none">
```

Comment:
The pie and bar charts do not resize appropriately on window resize. They look awesome when already loaded in smaller/bigger screen sizes though so nice one :+1: 

![image](https://user-images.githubusercontent.com/16066822/68163159-c7a72580-ffbe-11e9-9928-6e186e7a0143.png)

![image](https://user-images.githubusercontent.com/16066822/68163357-3b493280-ffbf-11e9-8ff4-7e1b065b3f99.png)